src/components/Home/NewPlayer.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import {useState} from 'react';
+import {Button, Input} from '@material-ui/core';
+
+interface IProps {
+	handleSubmit: Function;
+}
+
+export default function NewPlayer(props: IProps) {
+	const [name, setName] = useState('');
+
+	return(
+		<form onSubmit={(e) => {
```

Comment:
Any validation for duplicate\empty names?

Reply:
Fixed